### ISSUE TITLE
Plugin nativescript-google-maps-sdk is not included in preview app on device 26a7ab67-b5f6-4d4c-b38d-37c8fe0a528f and will not work.

### ISSUE THREAD

**Issue by krishna949** [dapriett/nativescript-google-maps-sdk#358]
i ma facing Plugin nativescript-google-maps-sdk is not included in preview app on device 26a7ab67-b5f6-4d4c-b38d-37c8fe0a528f and will not work.

**Comment by jlafitte** [comment]
I don't think this is a bug, it's just that preview only bundles a few plugins.  You'll need to use `tns run` to try it out on emulator or device.
